src/tabs/pill.scss

```diff
@@ -6,98 +6,54 @@
 
 @mixin iui-tabs-pill {
   @include iui-tabs;
-  display: flex;
   flex-flow: row;
   justify-content: space-evenly;
-  margin: $iui-baseline/2 0;
+  border-radius: $iui-border-radius;
+  gap: 2px;
```

Comment:
We avoid using gap for IE support

Reply:
Swapped `gap` with `margin-right: 2px` except on the last child.